File: gateway/gateway-api/src/main/java/ru/gilko/gatewayapi/constants/ControllerUrls.java
package ru.gilko.gatewayapi.constants;

public class ControllerUrls {
    private ControllerUrls() {
    }



    public static final String CARS_URL = "/cars";
    public static final String RENTAL_URL = "/rental";
    public static final String RENTAL_URL_WITH_ID = RENTAL_URL + "/{rentalUid}";
    public static final String RENTAL_URL_FINISH = RENTAL_URL_WITH_ID + "/finish";
    public static final String STATISTIC = "/statistic";
    public static final String CANCELLED = STATISTIC + "/cancelled";
    public static final String PROFITABLE = STATISTIC + "/profitable";
}

File: frontend/src/constants/ControllerUrls.js
const PROTOCOL = "http://"
const SERVER_URL = PROTOCOL + "127.0.0.1:8080"

const API_URL = SERVER_URL + "/api/v1"

export const CARS_URL = API_URL + "/cars";
export const RENTAL_URL = API_URL + "/rental";
export const RENTAL_URL_WITH_ID = (rentalUid) => RENTAL_URL + `/${rentalUid}`;
export const RENTAL_URL_FINISH = (rentalUid) => RENTAL_URL_WITH_ID(rentalUid) + "/finish";

export const STATISTIC = API_URL + "/statistic";
export const CANCELLING = STATISTIC + "/cancelled";
export const PROFITABLE = STATISTIC + "/profitable";